src/test/resources/pages/commonAssertRegexTestPage.html

```diff
@@ -1,13 +1,14 @@
 <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml"><head>
+<html xmlns="http://www.w3.org/1999/xhtml">
+   
 <head>
     <link href="/favicon.png" rel="shortcut icon" /> 
     <title>TEAMMATES - Instructor</title>
     <link type="text/css" href="/stylesheets/main.css" rel="stylesheet" />
     <link type="text/css" href="/stylesheets/evaluation.css" rel="stylesheet" />
 
     <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
-        <script src="/js/helper.js" language="JavaScript"></script>
+    <script src="/js/helper.js" language="JavaScript"></script>
```

Comment:
Unrelated change.

Reply:
@whipermr5   sir Actually I am going to attend the lecture can I perform the task after few hour .pardon me for delay .

Reply:
Sure. Actually you might want to open a new PR and close this one when you continue. @wkurniawan07 discovered that the CI build somehow cannot begin on this PR.

Reply:
Sure sir @whipermr5  can trailing white space can't be removed directly using edit option on github

Reply:
The GitHub editor can do it; just delete the spaces at the end of the line:
<img width="1006" alt="screen shot 2017-10-23 at 12 03 57 pm" src="https://user-images.githubusercontent.com/6811428/31871929-650feb82-b7ea-11e7-820b-2729a49d429d.png">